src/examples_types.jl

```diff
@@ -220,22 +220,19 @@ function get_state_restart(comms_ctx)
 end
 
 function get_initial_condition(parsed_args)
-    if is_baro_wave(parsed_args)
-        if parsed_args["moist"] == "dry"
-            return ICs.DryBaroclinicWave(parsed_args["perturb_initstate"])
+    if isnothing(parsed_args["turbconv_case"])
+        if parsed_args["initial_condition"] in ["DryBaroWave", "MoistBaroWave", "DecayingProfile"]
```

Comment:
```suggestion
        if parsed_args["initial_condition"] in ["DryBaroclinicWave", "MoistBaroclinicWave", "DecayingProfile"]
```

Reply:
Must have forgotten to push the fix to this! Thank you.